core/src/main/java/org/apache/iceberg/puffin/StandardBlobTypes.java

```diff
@@ -26,4 +26,6 @@ private StandardBlobTypes() {}
    * href="https://datasketches.apache.org/">Apache DataSketches</a> library
    */
   public static final String APACHE_DATASKETCHES_THETA_V1 = "apache-datasketches-theta-v1";
+
+  public static final String PRESTO_SUM_DATA_SIZE_BYTES_V1 = "presto-sum-data-size-bytes-v1";
```

Comment:
we don't need to store the exact parameter used by presto as part of iceberg.
we can use it in the test or even use a dummy  identifier to simulate the existence of additional non supported metadata.

separately we should reach agreement on what is the right way to store the data size in the puffin file cross engines.

Reply:
I think we should remove this since Iceberg doesn't support this yet

Reply:
+1, it shouldn't be here. If it is a generic blob type we want to support across engines, we should discuss this on the dev list and vote.